models/comments-model.js

```diff
@@ -0,0 +1,24 @@
+const db = require("../db/connection");
+exports.selectCommentsByArticleId = (params) => {
+  const id = params.article_id;
+  if (isNaN(id)) {
+    return Promise.reject({ msg: "Bad Request", code: 400, custom: true });
+  }
+
```

Comment:
You don't need to check whether the id is a number or not, if you try to query the database for an article_id that is not a number, the query will fail and PSQL will throw an error for you. You can catch this in the controller and deal with a err.code.